python/mlcroissant/mlcroissant/_src/structure_graph/base_node.py

```diff
@@ -178,9 +177,16 @@ def validate_name(self):
             self.add_error(
                 f'The identifier "{name}" is too long (>{_MAX_ID_LENGTH} characters).'
             )
-        regex = re.compile(rf"^{ID_REGEX}$")
+        if self.ctx.is_v0():
```

Comment:
`ctx.is_v0` is likely to appear many times in the code, before we do the refacto of a config-based parsing of the JSON-LD. So I think it should be explicit without needing a comment each time we use it. What would you change to make it more explicit while still readable/short?

Reply:
Maybe just a comment line above? Not sure though, maybe it is unnecessary. You call :)

Reply:
Acknowledge.

I would rather change everywhere `ctx.is_v0` is used by a more explicit wording. Otherwise, we'd have to add a comment at each occurrence. We can think of it and do the change in another PR.